config/checker/javax.swing.astub

```diff
@@ -24,5 +24,4 @@ class JTable {
   TableModel createDefaultDataModel(@UnknownInitialization JTable this);
   void setModel(@UnknownInitialization JTable this, TableModel dataModel);
   void initializeLocalVars(@UnknownInitialization JTable this);
-  void updateUI(@UnknownInitialization JTable this);
```

Comment:
Hmm I think `updateUI` can/should stay `@UnknownInitialization this`... whether we're going to override or not :/ It's just annotated due to the fact that it's invoked by `JTable`'s c'tor